src/test/kotlin/com/pubnub/api/subscribe/eventengine/effect/BaseEffectTest.kt

```diff
@@ -5,8 +5,8 @@ import com.pubnub.api.models.consumer.pubsub.BasePubSubResult
 import com.pubnub.api.models.consumer.pubsub.PNEvent
 import com.pubnub.api.models.consumer.pubsub.PNMessageResult
 
-open class BaseEffectTest {
-    protected fun createMessages(): List<PNEvent> {
+abstract class BaseEffectTest {
```

Comment:
Do we need a class at all? Why not just make the `createPNMessageResultList` just a function?

Reply:
I wanted do have function that will be used by many classes.
Is static method better design that having abstract class with method like here?